taxi/views.py

```diff
@@ -87,3 +98,38 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+
+    def get_success_url(self):
+        return reverse_lazy(
+            "taxi:driver-detail",
+            kwargs={"pk": self.object.pk}
+        )
+
+
+class UpdateDriverView(View):
+    @staticmethod
+    def post(request, pk):
```

Comment:
use self.request instead of request and delete `@staticmethod`
```suggestion
    def post(self, pk):
```

Reply:
when I write like that, - "def post(self, pk):" -  I get an error
[TypeError: products() got multiple values for argument 'pk']

works when I write like this - "def post(self, request, pk):"